src/main/java/com/boardgo/domain/mapper/UserInfoMapper.java

```diff
@@ -0,0 +1,14 @@
+package com.boardgo.domain.mapper;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+import com.boardgo.domain.user.controller.dto.SignupRequest;
+import com.boardgo.domain.user.entity.UserInfoEntity;
+
+@Mapper
+public interface UserInfoMapper {
+	UserInfoMapper INSTANCE = Mappers.getMapper(UserInfoMapper.class);
+
+	UserInfoEntity from(SignupRequest signupRequest);
```

Comment:
MapStruct의 메소드 네이밍을 `to***Dto` `to***Entity` 로 설정하기로 얘기했던 것 같아서

`toUserInfoEntity` 로 변경하는게 어떨까용?
```suggestion
	UserInfoEntity toUserInfoEntity(SignupRequest signupRequest);
```

Reply:
헛 .. 모두 변경해야겠네요 ㅠㅠ